proxy.php

```diff
@@ -50,7 +47,7 @@ public function __construct()
 		global $image_proxy_enabled, $image_proxy_maxsize, $image_proxy_secret, $cachedir, $sourcedir;
 
 		require_once(dirname(__FILE__) . '/Settings.php');
-		require_once($sourcedir . '/Class-CurlFetchWeb.php');
+		require_once($sourcedir . '/Subs.php');
```

Comment:
I like to point out,
that the Subs,php is garbage for the default behavior,
only in the special case when curl is not avaible (which is rar) the subs are needed,
for this load on every call this garbage (~5k lines) is useless.